### ISSUE TITLE
SystemPackageTool update() not working with yum

### ISSUE THREAD

**Issue by SirLemyDanger** [conan-io/conan#1599]
I get the follwing error on CentOS 7 / python 2.7.5 / conan 0.25 when I use

```from conans import ConanFile, CMake
from conans.tools import os_info, SystemPackageTool

class someConan(Conanfile):
...
    def system_requirements(self):
        installer = SystemPackageTool()
        installer.update()
```

> PROJECT: ERROR: while executing system_requirements(): Command 'sudo yum check-update' failed
ERROR: Error in system requirements

I assume that conan handles the exit code of `yum check-update` incorrect. For whatever reason yum decided to exit with **exitcode 100** if there are updates available.

**Comment by lasote** [comment]
Thanks for reporting!

**Comment by memsharded** [comment]
Fixed, will be released in 0.26
